libraries/standard/mqtt/include/mqtt_lightweight.h

```diff
@@ -327,25 +334,38 @@ MQTTStatus_t MQTT_GetSubscribePacketSize( const MQTTSubscribeInfo_t * pSubscript
 /**
  * @brief Serialize an MQTT SUBSCRIBE packet in the given buffer.
  *
+ * #MQTT_GetSubscribePacketSize should be called with @p pSubscriptionList
```

Comment:
#MQTT_GetSubscribePacketSize should be called with @p pSubscriptionList before invoking this function to get the size of the required MQTTFixedBuffer_t  and remainingLength. The MQTTFixedBuffer_t  must be at least as largse as the size returned by MQTT_GetSubscribePacketSize and the remainingLength must be as returned by MQTT_GetSubscribePacketSize ,